docs/builds.rst

```diff
@@ -46,7 +46,7 @@ we will first look for a ``mkdocs.yml`` file in the root of your repository.
 If we don't find one,
 we will generate one for you.
 
-Then MkDocs will build any files with a ``.md`` extension.
+Then MkDocs will build any files with a ``.md`` extension within a single directory called ``docs``, ``doc``, ``Doc``, or ``book`` (this is the order in which directories are searched).
```

Comment:
I think this should be a little more clear since rtd only search in this directories if it doesn't find a mkdocs.yml file

Reply:
Got it. I can add a bit more detail to make the entire process more clear.